recipes/kmedoids/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "kmedoids" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kno10/python-kmedoids/archive/refs/tags/v{{ version }}.tar.gz
+  #  `openssl sha256 <file name>`.
+  sha256: d334848ecc0d4b17c0e041b975f29b1963cd64e7c3743d9b9ec157ae5780e2c2
+
+build:
+  number: 0
+  script:
+    - export CARGO_NET_GIT_FETCH_WITH_CLI=true
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - cargo-bundle-licenses
+    - numpy
+  host:
```

```suggestion
  host:
    - numpy
```

Typically numpy is a host dependency, not a build dependency. If this doesn't work, you should really explain why it is critical to have numpy in the build section requirement (sometimes this is required for cross compilation).